packages/core/database/lib/entity-manager/index.js

```diff
@@ -592,7 +661,7 @@ const createEntityManager = (db) => {
             })
             .execute();
 
-          const rows = toAssocs(data[attributeName]).map((data) => ({
+          const rows = cleanRelationData.set.map((data) => ({
```

Comment:
I think you should check if cleanRelationData is empty.
It is giving me an error if it is (trying to update a relation from a kitchensink type in getstarted)

Reply:
Indeed 👌  I'll fix that when adding connect/disconnect support to morph relations